experimental/torchprep/torchprep/pruning.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+
+import torch
+import torch.nn.utils.prune
+import typer
+
+from .format import Device
+from .utils import load_model
+
+
+def _prune(
```

Comment:
Do we have any numbers on the perf of pruning?

Reply:
I don't, anecdotally performance can be worst if you don't have proper hardware support for pruning